src/NucleiEnv.php

```diff
@@ -64,37 +61,33 @@ public function __construct(
      */
     public function prepareCommand(
         string $spEntityId,
-        string $target,
         string $ascUrl,
         string $token,
         string $testId = null,
     ): string {
         $spTestResultsDir = $this->getSpTestResultsDir($spEntityId);
-        $fileName = $this->helpers->filesystem()->cleanFilename($spEntityId);
+        $screenshotsDir = escapeshellarg(rtrim($spTestResultsDir . '/') . '/pictures/');
 
         // Escape shell args.
         $spEntityId = escapeshellarg($spEntityId);
         $target = escapeshellarg($target);
         $acsUrl = escapeshellarg($ascUrl);
-        $token = escapeshellarg($token);
+        $authorization = escapeshellarg("Bearer $token");
         $testId = empty($testId) ? null : escapeshellarg($testId);
 
         // First use the raw HTTP template to run the tests.
         $this->templateId = self::NUCLEI_TEMPLATE_SAML_RAW_ALL;
 
         $command =
             "mkdir -p $spTestResultsDir; " .
-            "nuclei -target $target " .
+            "nuclei -target $acsUrl " .
             "-env-vars -headless -matcher-status -follow-redirects -disable-update-check -timestamp " .
             "-no-mhe -restrict-local-network-access -dialer-keep-alive 30 -dialer-timeout 30 " .
             "-templates {$this->getTemplatesPath()} " .
             "-var SP_ENTITY_ID=$spEntityId " .
-            "-var CONSUMER_URL=$acsUrl " .
             "-var CONFORMANCE_IDP_BASE_URL=$this->conformanceIdpBaseUrl " .
-            "-var CONFORMANCE_IDP_HOSTNAME=$this->conformanceIdpHostname " .
-            "-var RESULT_OUTPUT_DIR=$spTestResultsDir " .
-            "-var FILENAME=$fileName " .
-            "-var TOKEN=$token " .
+            "-var RESULT_OUTPUT_DIR=$screenshotsDir " .
```

Comment:
How about we then change the RESULT_OUTPUT_DIR to be more intuitive, like SCREENSHOTS_DIR or similar.

Reply:
will do